scripts/release-postinstall.js

```diff
@@ -0,0 +1,60 @@
+/**
```

Comment:
@andreypopp - I factored this out to a separate file from the `Makefile` since the logic grew a bit. Let me know if you have a preference either way, though.